projects/BFDR/IJENatality.cs

```diff
@@ -28,8 +28,40 @@ public abstract class IJENatality : IJE
         // Class helper methods for getting and settings IJE fields.
         //
         /////////////////////////////////////////////////////////////////////////////////
+        
+
+        ///
+        /// <summary>Sets FirstName.</summary>
+        /// 
+        protected string[] Update_FirstName(string value, string[] givenNames)
```

Comment:
These helper functions should be moved to VitalRecord_util with the other helper functions that were added in this PR.

Reply:
I take that back, since they are only used in IJE they do make sense here